packages/preset-anchor/src/generators/application/application-anchor-schema.d.ts

```diff
@@ -1,8 +1,19 @@
-export type ApplicationAnchorTemplate = 'counter' | 'hello-world'
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
 export interface ApplicationAnchorSchema {
-  name: string
-  skipFormat?: boolean
-  template?: ApplicationAnchorTemplate
+  name: string;
```

Comment:
The `name` arg for most of the generators is not adding the description into the generated schema.d.ts files. 
It looks like the `name` property for each of the 3 current presets do not have a description set. Is there a reason for that? (they do have a prompt text, just not description)

Reply:
Just added the description to the schema.json files and now they get added